en/docs/Administer/ProductAdministration/Multitenancy/managing-tenants.md

```diff
@@ -18,37 +18,50 @@ You can add a new tenant in the management console and then view it by following
-2.  Click the **Visit Store** link associated with a given store to open it.
-3.  Anonymous users can browse all stores and all public APIs that are published to them. However, in order to subscribe to an API, the user must log in.
+1.  Access the API Developer Portal URL (by default, `https://localhost:9443/devportal` ) using a Web browser. You see the storefronts of all the registered tenant domains listed there. For example,
+    [![](../../../../assets/img/Administer/tenant-developer-portals.png)](../../../../assets/img/Administer/tenant-developer-portals.png)
+    
+     This is called the public Developer Portal. Each icon here is linked to the API Developer Portal of a registered tenant, including the super tenant, which is `carbon.super` . That is, the super tenant is also considered a tenant.
+
+2.  Click the domain associated with a given Developer Portal to open it.
+
+3.  Anonymous users can browse all Developer Portals and all public APIs that are published to them. However, in order to subscribe to an API, the user must log in.
 
     For example, if you are a user in the `domain1.com` tenant domain,
 
-    -   You can access the public store ( `https://localhost:9443/store` ), go to the `domain1.com` store, log in to it and subscribe to its APIs.
-    -   You can also browse the other tenant stores listed in the public store. But, within other tenant stores, you can only subscribe to the APIs to which your tenant domain is permitted to subscribe to. At the time an API is created, the API creator can specify which tenants are allowed to subscribe to the API. For information, see [API Subscriptions](https://docs.wso2.com/display/AM260/Create+and+Publish+an+API#CreateandPublishanAPI-Subscriptions) .
+    -   Click the **Go To Public Dev Portal** option available in the Navigation bar, go to the `domain1.com` Developer Portal, log in to it and subscribe to its APIs.
+    [![](../../../../assets/img/Administer/tenant-naviagtion-bar.png)](../../../../assets/img/Administer/tenant-naviagtion-bar.png)
+    -   You can also browse the other tenant Developer Portals listed in the public Developer Portal. But, within other tenant Developer Portals, you can only subscribe to the APIs to which your tenant domain is permitted to subscribe to. At the time an API is created, the API creator can specify which tenants are allowed to subscribe to the API. For information, see [API Subscriptions](../../../Learn/ConsumeAPI/ManageSubscription/subscribe-to-an-api.md) .
+
+    !!! info
+        Deleting tenant which is not available in the Management Console UI, can be done through the `RemoteTenantManager` Admin Service. You can invoke these operations using a SOAP client like SOAP UI. Follow the steps below to do the onfigurations using SOAP UI.
```

Comment:
A tenant can be deleted through the `RemoteTenantManager` Admin Service. Admins service oprations can be invoked using a SOAP client such as SOAP UI.
Follow the steps below to do the onfigurations using SOAP UI.